src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -179,6 +179,35 @@ static void uct_rc_mlx5_iface_progress_enable(uct_iface_h tl_iface, unsigned fla
                                       iface->super.progress, flags);
 }
 
+static ucs_status_t uct_rc_mlx5_iface_create_qp(uct_ib_iface_t *iface,
+                                                uct_ib_qp_attr_t *attr,
+                                                struct ibv_qp **qp_p)
+{
+#if HAVE_DECL_MLX5DV_CREATE_QP
+    uct_ib_device_t *dev               = uct_ib_iface_device(iface);
+    struct mlx5dv_qp_init_attr dv_attr = {};
+    struct ibv_qp *qp;
+
+    uct_ib_iface_fill_attr(iface, attr);
+#if HAVE_DECL_MLX5DV_QP_CREATE_ALLOW_SCATTER_TO_CQE
+    dv_attr.comp_mask    = MLX5DV_QP_INIT_ATTR_MASK_QP_CREATE_FLAGS;
+    dv_attr.create_flags = MLX5DV_QP_CREATE_ALLOW_SCATTER_TO_CQE;
+#endif
+    qp = mlx5dv_create_qp(dev->ibv_context, &attr->ibv, &dv_attr);
+    if (qp == NULL) {
+        ucs_error("iface=%p: failed to create QP: %m", iface);
+        return UCS_ERR_IO_ERROR;
+    }
+
+    attr->cap = attr->ibv.cap;
+    *qp_p     = qp;
+
+    return UCS_OK;
+#else
```

Comment:
do we set get.min_zcopy to 0, for uct_iface_query(), in case HAVE_DECL_MLX5DV_CREATE_QP is not defined?

Reply:
@Artemy-Mellanox let's also assert we do not get inline CQE if we had set get.min_zcopy to 0 (during poll_tx)

Reply:
use uct_rc_txqp_completion_op to determine if the operation was get_zcopy